Assets/Scripts/UI/UIView.cs

```diff
@@ -29,10 +30,10 @@ private void Start()
         public void IncrementScore(int scoreToIncrement)
         {
             currentScore += scoreToIncrement;
-            scoreText.SetText(currentScore.ToString());
+            scoreText.text = currentScore.ToString();
```

Comment:
technically there is no difference here in both the approaches. But we will always prefer the one which is more readable in general. Hence we will be using SetText() method although internally its the same thing.